docs/1.0. Certificate Provisioning.md

```diff
@@ -68,46 +68,48 @@ The entity that is using the NMOS API, for example:
-    - The EST Server will use the TLS Client Certificate to authenticate and then authorise whether the EST Client. If the EST Client meets the local CA's security policy, it will be issued with a TLS Server Certificate.
-6. The EST Client should provide the returned TLS Certificate and the corresponding chain of trust in all future responses returned by its NMOS APIs
+1. Before the NMOS Server (EST Client) is shipped from the factory it must be provisioned with a unique TLS Client Certificate, signed by the manufacturer's Certificate Authority.
+2. When the EST Client is connected to the target environment's network, it will first discover the location of the EST Server using unicast DNS-SD.
+3. The EST Client should(?) then request the Root CA for the target network, from the EST Server.
+   - The EST Client should(?) assume the EST server found using DNS-SD is trusted and need not perform TLS authentication of the EST Server for this initial transaction.
+   - It uses the Root CA returned to secure further communications with the EST Server and (?) NMOS Servers.
+4. The EST Client must generate a Certificate Signing Request (CSR) for all supported digital signature algorithms with appropriate key lengths.
+   - Included in the CSR must be the DNS resolvable name for the current domain of the EST Client.
+   - Appropriate values for the remaining CSR fields should(?) be used.
+5. The EST Client should(?) send each CSR to the EST Server.
+   - The EST Client must(?) provide the manufacturer-issued certificate during the TLS handshake, if this certificate is valid.
+   - The EST Server will use the TLS Client Certificate to authenticate and then authorise whether the EST Client (what?). If the EST Client meets the local CA's security policy, it will be issued with a TLS Server Certificate. (How come RFC 7030 doesn't mention server certs??)
+6. The EST Client should(?) provide the returned TLS Certificate and the corresponding chain of trust in all future responses returned by its NMOS APIs.
 
-The workflow if no valid manufacturer TLS Client Certificate is present or if EST Client is moved to a new network are covered in detail in the following [sections](#est-client).
+The workflow if no valid manufacturer TLS Client Certificate is present or if the EST Client is moved to a new network are covered in detail in the following [sections](#est-client).
 
-![NMOS EST WorkFlow](images/nmos_est_workflow.png)
+![NMOS EST Workflow](images/nmos_est_workflow.png)
+(Why "Node" in this diagram?)
 
 ## DNS-SD Advertisement
 
 The EST Server MUST be advertised using unicast DNS-SD as per [RFC 6763][RFC-6763].
 The EST Server SHOULD NOT be advertised via mDNS-based DNS-SD.
-EST Clients MUST not trust a mDNS-based DNS-SD advertisement for the EST server unless explicitly configured to do so.
+EST Clients MUST NOT trust an mDNS-based DNS-SD advertisement for the EST server unless explicitly configured to do so.
```

Comment:
I feel the "mDNS-based DNS-SD" is a little confusing... does this alter the meaning?

```suggestion
EST Clients MUST NOT trust an mDNS advertisement for the EST server unless explicitly configured to do so.
```